StalkStocks/StalkStocks
# set up chart.jsx and initialized the chart with some mock data.

Description
this add the Chart component which takes data and will add the graph to the dom. currently has mock data to simulate the data we retrieve from our api.

Asana Task
N/A

Testing Instructions
Run npm install to install dependancies, npm build to set up dist folder, and npm start to spin up server. Localhost 3000 and 8000 will run on npm run dev as well. on the port you ran the front end, check to see if the chart is rendering.

Checklist
Please go through each item of this checklist carefully.

Additional Notes, Images, etc.
N/A
